integration/spark2/src/main/scala/org/apache/spark/sql/parser/CarbonSpark2SqlParser.scala

```diff
@@ -121,6 +126,15 @@ class CarbonSpark2SqlParser extends CarbonDDLSqlParser {
         AlterTableCompactionCommand(altertablemodel)
     }
 
+  protected lazy val createDataMap: Parser[LogicalPlan] =
```

Comment:
Can you add comment to describe the syntax

Reply:
ok